.circleci/config.yml

```diff
@@ -241,13 +237,37 @@ jobs:
   # This helps ensure the overall stability of the plugin prior to publishing.
   linux-tarball-test:
     <<: *defaults
+    parameters:
+      node-version: &node_version_param # Define an anchor here, since we'll want to use this parameter again elsewhere.
+        type: enum
```

Comment:
We're using an enum-type parameter instead of a boolean, because using a matrix with this parameter causes its value to be appended to each created job, and `linux-tarball-test-lts` is more immediately meaningful than `linux-tarball-test-true`.

Reply:
@jfeingold35 could you leave this comment in the code as well? Would be helpful when a different person tries to read the code in a few months.